_layouts/posts.html

```diff
@@ -38,4 +46,5 @@
         </div>
     </div>
 {% endfor %}
-</div>
\ No newline at end of file
+</div>
+<script src="/js/posts.js"></script>
```

Comment:
It might be good to put the script directly in the `_layouts/base.html` with the other js scripts (with a if checking if name of the page is post)

Reply:
Ok, I will add the js script with the others.